.github/workflows/quarkus-snapshot.yaml

```diff
@@ -16,9 +16,6 @@ jobs:
     if: github.actor == 'quarkusbot'
 
     steps:
-      - name: Install yq
-        run: sudo add-apt-repository ppa:rmescandon/yq && sudo apt update && sudo apt install yq -y
-
```

Comment:
This is included in Ubuntu runners now and it's causing issues with Ubuntu 24 as this repository is discontinued.